marklogic-data-hub-central/ui/src/components/modeling/entity-type-table/entity-type-table.tsx

```diff
@@ -23,15 +24,24 @@ const EntityTypeTable: React.FC<Props> = (props) => {
       title: 'Name',
       dataIndex: 'name',
       width: 400,
-      // TODO - Edit Entity DHFPROD-4452
       render: text => {
+        let parseText = text.split(',');
+        let entityName = parseText[0];
+        let entityDescription = parseText[1];
+
         return (
           <Tooltip title={ModelingTooltips.entityTypeName}>
-            <span data-testid={text + '-span'}>{text}</span>
+            <span data-testid={text + '-span'}
```

Comment:
Please add CSS to make is look similar to the instance link. ( color change, cursor: pointer )

Reply:
Updated in here: https://github.com/marklogic/marklogic-data-hub/pull/4017/commits/e8d869cc937370f183c2574b9e61293edd943442